log-analyzer/features/org.wso2.analytics.common.la.apim.feature/src/main/resources/gadgets/LogErrorFilteredMessage/js/main.js

```diff
@@ -15,15 +15,17 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-var gatewayPort = location.port -9443 + 8243; //Calculate the port offset based gateway port.
-var serverUrl = "https://"+location.hostname +":"+ gatewayPort+"/LogAnalyzerRestApi/1.0";
+var gatewayPort = location.port - 9443 + 8243; //Calculate the port offset based gateway port.
```

Comment:
@ksdperera In a lot of places pf log analyser code this calculation has been done. Can't we implement this in a common util JS file and reuse it? else if there are improvements then we need to change everywhere.